faiss/gpu/CMakeLists.txt

```diff
@@ -304,6 +318,12 @@ foreach(header ${FAISS_GPU_HEADERS})
   )
 endforeach()
 
+if (USE_ROCM)
+find_package(HIP REQUIRED)
+find_package(hipBLAS REQUIRED)
+target_link_libraries(faiss_gpu PRIVATE hip::host roc::hipblas)
+target_compile_options(faiss_gpu PRIVATE)
+else()
```

Comment:
1. indentation
2. Can we please move `find_package` statements to the top-level `CMakeLists.txt` guarded by `USE_ROCM`?
3. For `target_link_libraries`, can we please use the condition syntax like we do for RAFT? `$<$<BOOL:${USE_ROCM}>:hip::host>`